parsl/tests/test_mpi_apps/test_mpiex.py

```diff
@@ -43,16 +43,17 @@ def test_init():
     """Ensure all relevant kwargs are copied over from HTEx"""
 
     new_kwargs = {'max_workers_per_block'}
-    excluded_kwargs = {'available_accelerators', 'enable_mpi_mode', 'cores_per_worker', 'max_workers_per_node',
-                       'mem_per_worker', 'cpu_affinity', 'max_workers'}
+    excluded_mpix_kwargs = {'available_accelerators', 'enable_mpi_mode', 'cores_per_worker', 'max_workers_per_node',
+                            'mem_per_worker', 'cpu_affinity', 'max_workers'}
+    excluded_htex_kwargs = {'manager_selector'}
```

Comment:
what are you claiming about behaviour with this?

before this PR, excluded_kwargs was used as part of assertions like:
"the mpi executor args must look something like:  the htex args minus the excluded_kwargs plus the new_args"

(so excluded_args is the arguments htex should have that mpix does not have, and new_kwargs is the arguments mpix should have that htex does not have)

so whats being implemented by this test you could visualise as a big venn diagram of who has which kwargs:

one circle is the htex args, one circle is the mpix args, with most args being in the central overlap, and new_kwargs and excluded_kwargs being the two side bits.

with that as the basic test model, what are you expressing by adding in excluded_htex_kwargs?

Reply:
That's on me. It wasn't clear to me whether excluded_args were the extra args for HTEX or mpi executor. Reverting changes and adding "manager_selector" to excluded_args.